rebar.config.script

```diff
@@ -50,7 +50,7 @@ CFlags =
         {"priv/esqlite3_nif.so", NifSources}
     ]},
 
-    {plugins, [pc]},
+    {plugins, [{pc, "~> 1.15"}]},
```

Comment:
I have no clue why this is required to use 1.15 over 1.14 for me locally.

Reply:
I don't know either... We'll see what it does. 👍🏻